bower.json

```diff
@@ -28,6 +28,7 @@
     "purescript-maybe": "~0.3.0"
   },
   "devDependencies": {
-    "purescript-console": "~0.1.0"
+    "purescript-console": "~0.1.0",
+    "purescript-assert": "master"
```

Comment:
Can you bump this to ~0.1.0? It doesn't exist yet, but then none of the versions we're referencing in the revised libraries do yet :wink: